oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/mongo/MongoDocumentStore.java

```diff
@@ -2111,6 +2115,11 @@ public <T extends Document> T getIfCached(Collection<T> collection, String key)
         return doc;
     }
 
+    @Override
+    public boolean isCommitCleanupFeatureEnabled() {
+        return docStoreCommitCleanupFeature != null && docStoreCommitCleanupFeature.isEnabled();
```

Comment:
I would this small logic to Utils class, making it easier for writing unit cases as well.

Reply:
Yep